lib/Basic/CmdLineOptions.cpp

```diff
@@ -96,14 +105,6 @@ void KCommandLine::HideUnrelatedOptions(cl::OptionCategory &Category) {
   }
 }
 
-void KCommandLine::HideUnrelatedOptions(
-    ArrayRef<const cl::OptionCategory *> Categories) {
-  for (ArrayRef<const cl::OptionCategory *>::iterator i = Categories.begin(),
```

Comment:
Why do remove this part?

Reply:
This was accidentally removed during the last rebase, we added it back in.